feature-group-definitions/starting-style.dist.yml

```diff
@@ -9,8 +9,6 @@ status:
     chrome: "117"
     chrome_android: "117"
     edge: "117"
-    safari: "17.5"
-    safari_ios: "17.5"
```

Comment:
@foolip This introduces a potentially undesirable oddity: we will no longer see version info for future-dated releases. I think this is correct, but it is a difference from before.

Reply:
Ah, I didn't realize 17.5 was a future release. I agree that's the right change though.